src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala

```diff
@@ -42,10 +42,10 @@ import scala.util.{ Failure, Success, Try }
  */
 package object servlets extends DebugEnhancedLogging {
 
-  private val extensionZipPattern = ".*.g?z(ip)?"
+  val extensionZipPattern = ".*.g?z(ip)?"
```

Comment:
this is of course a regex. Let me see if I understand this one correctly. The following should all match **the latter part** of this pattern:

* ".gzip"
* ".gz"
* ".zip"
* ".z"

especially the last one is one that I'm doubting this pattern should match. Am I overlooking something?

Reply:
It might have been unintentionally but now that you make me look again... We recently discussed `tar` but there is also `rar` and and a lot more exotic variations.

It is more in the scope of [EASY-2028](https://drivenbydata.atlassian.net/browse/EASY-2028) to check which types we support.

https://www.file-extensions.org/z-file-extension
https://www.file-extensions.org/filetype/extension/name/archive-and-compressed-files

Reply:
The documentation only talks about the content-type. Should we look at the extension anyway? Again more in the scope of EASY-2028.

https://dans-knaw.github.io/easy-deposit-api/api.html#/directory/post_deposit__id__file__dir_path_

Reply:
A unit test shows the .z was intentional
https://github.com/DANS-KNAW/easy-deposit-api/blob/7f03c5d174a6970b089db7f90b8efb18c90908a3/src/test/scala/nl.knaw.dans.easy.deposit/servlets/RichFileItemsSpec.scala#L52-L69